duckdb/README.md

```diff
@@ -1,6 +1,26 @@
 # DuckDB Data Connector
 
-Follow these steps to get started with DuckDB (v0.10.1) as a Data Connector.
+Follow these steps to get started with DuckDB (v0.10.2) as a Data Connector.
```

Comment:
I don't think we require specific DuckDB version to create a local database and it will also be hard to maintain this page moving forward (maintain latest version) so let's remove `(v0.10.2)` here